src/components/Badge/Badge.scss

```
@@ -0,0 +1,251 @@
+$font-color-white:var(--du-badge-white,#ffffff);
+$font-color-gray:var( --du-text-general,#868686);
+$success_color_border: var(--du-badge-success, #4bb4b3);
+$warning_color_border: var(--du-badge-warning, #ffad3a);
+$white_color: linear-gradient(#e6e9e9,#f2f3f4);
+$success_color: $gradient-color-blue-5;
+$warning_color: $gradient-color-yellow;
+$danger_color: var(--du-badge-danger, #ec3f3f);
+$blue_color: var(--du-badge-blue, #115BFF);
+$purple_color: var(--du-badge-purple, #722FE4);
+$light_blue_color: var(--du-badge-light-blue, #0DC2E7);
+$gold_color:$gradient-color-gold;
+$gold_color_border:$color_yellow;
+$green_color:$gradient-color-green-4;
+$green_color_border:$color-green-6;
+
+.deriv-badge {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+
+    &__size {
+        &--tight {
+            &.deriv-badge__padding--xs {
+                width: 34px;
+                height: 14px;
```

@yaswanth-deriv, do you guys (`@deriv/ui`) typically work with `px` as units for such? 🤔 
```suggestion
                width: 34px;
                height: 14px;
```

---

Yes, we are consistently using px throughout the library